pkg/result/processors/sort_results.go

```diff
@@ -142,6 +200,94 @@ func (cmp ByColumn) Compare(a, b *result.Issue) compareResult {
 	return res
 }
 
+func (cmp *byColumn) String() string {
+	return comparatorToString("byColumn", cmp)
+}
+
+type byLinter struct{ next comparator }
+
+func (cmp *byLinter) Next() comparator { return cmp.next }
+
+func (cmp *byLinter) AddNext(c comparator) comparator {
+	cmp.next = c
+	return cmp
+}
+
+func (cmp *byLinter) Compare(a, b *result.Issue) compareResult {
+	var res compareResult
+
+	if res = compareResult(strings.Compare(a.FromLinter, b.FromLinter)); !res.isNeutral() {
```

Comment:
too late, but I feel that code could be easier, because there is copypasting of `!res.isNeutral` && `cmp.Next()` logic in every comparator

looks like simple loop on `Process` level:

```go
for _, cmp := range cmps {
    if res := cmp.Compare(); !res.isNeutral() {
         return res
    }
}
```

and comparators became lightweighted:

```go
func (cmp *byLinter) Compare(a, b *result.Issue) compareResult {
    return strings.Compare(a.FromLinter, b.FromLinter)
}
```